pom.xml

```diff
@@ -30,12 +30,15 @@
   <url>https://github.com/aws/event-ruler/</url>
 
   <properties>
-    <jackson.version>2.13.3</jackson.version>
+    <jdk.version>1.8</jdk.version>
+    <jackson.version>2.13.4</jackson.version>
     <jsr.version>3.0.2</jsr.version>
     <junit.version>4.13.2</junit.version>
     <compiler.plugin.version>3.10.1</compiler.plugin.version>
     <javadoc.plugin.version>3.4.1</javadoc.plugin.version>
     <source.plugin.version>3.2.1</source.plugin.version>
+    <gpg.plugin.version>3.0.1</gpg.plugin.version>
+    <staging.plugin.version>1.6.8</staging.plugin.version>
```

Comment:
Used to be 1.6.13. Is the change intentional?

Reply:
Yes for now. I got publish to maven working when doing the version bump of this pkg to 1.0.2. I cannot test if bumping it back to 1.6.13 will work until I do another version bump for , so will be doing that in a follow up commit where I change the version to 1.0.3

Reply:
https://github.com/aws/event-ruler/pull/48